content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,413 @@
+---
+title: 'Metrics'
+metaTitle: 'Metrics'
+metaDescription: 'Diagnose application performance with insights into Prisma Client database activity'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma metrics give you a detailed insight into how Prisma Client interacts with your database. You can use this insight to help diagnose performance issues with your application.
+
+</TopBlock>
+
+## About metrics
+
+You can export metrics in JSON or Prometheus formats and view them in a console log, or integrate them into an external metrics system, such as StatsD or Prometheus. If you integrate them into an external metrics system, then you can view the metrics data over time. For example, you can use metrics to help diagnose how your application's number of idle and active connections changes.
+
+We provide the following metrics:
+
+- Counters (always increase):
+
+  - `query_total_queries`: The total number of database queries executed.
+  - `query_total_operations`: The total number of Prisma operations executed.
+    - A Prisma operation is a statement like `let user = prisma.user(include: {posts: true}})`, which Prisma Client translates into one or more SQL queries.
+    - The value returned by `query_total_operations` can be less than `query_total_queries`, because some operations create multiple queries.
+
+- Gauges (can increase or decrease):
+  - `pool_active_connections`: The number of active connections in use.
+  - `pool_idle_connections`: The number of connections that are not being used.
+  - `pool_wait_count`: The number of workers waiting for a connection.
+  - `query_active_transactions`: The number of active transactions.
+- Histograms (distributions of values categorized into "buckets"):
```

Comment:
Need a blank space between Gauges and Histograms, and consistent space (or not) between header and bullet items.